Resources/config/serializer/PhpcrOdmResource.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<serializer>
+    <class name="Symfony\Cmf\Component\Resource\Repository\Resource\PhpcrOdmResource" xmlns:h="https://github.com/willdurand/Hateoas">
+        <h:relation rel="self">
+            <h:href route="_cmf_resource">
+                <h:parameter name="repositoryName" value="testrepo" />
+                <h:parameter name="path" value="expr(object.getPath())" />
+            </h:href>
+        </h:relation>
+        <virtual-property serialized-name="children" max-depth="0" method="listChildren">
+            <xml-list inline="true" entry-name="foobar" />
+        </virtual-property>
```

Comment:
For some reason I get the following .. i.e. a "resources" key which seems redundant. @WouterJ, @lsmith77 is this what you would expect?

````json
"children": {
                    "resources": {
                        "foo": {
                            "_links": {
                                "self": {
                                    "href": "/api/testrepo//foo/bar/foo"
                                }
                            },
                            "children": {
                                "resources": []
                            },
                            "document": {
                                "body": "This is my article",
                                "id": "/tests/cmf/articles/foo/bar/foo",
                                "title": "Article 1"
                            },
                            "path": "/foo/bar/foo",
                            "repo_path": "/foo/bar/foo"
                        }
                    }
                },
````